### PR TITLE
Updated keypoints method in Vis

### PR DESCRIPTION
**Pull request by acpaquette**
It seems like someone already did the majority of these but missed one in the process. Either that or I am looking at the wrong file.

### CODE REVIEW

**Comment by Kelvinrr** [comment]
Yeah, Jay got most of them in #94

File: autocnet/vis/graph_view.py [hunk 1]
```diff
@@ -94,7 +94,7 @@ def plot_node(node, ax=None, clean_keys=[], **kwargs):
     keypoints = node.get_keypoints()
     if clean_keys:
         matches, mask = node._clean(clean_keys)
-        keypoints = node.keypoints[mask]
+        keypoints = node.get_keypoints[mask]
```
Comment (Kelvinrr) [inline autocnet/vis/graph_view.py]: Isn't the proper use here is `get_kepoints(index = mask)`?

Comment (jlaura) [inline autocnet/vis/graph_view.py]: `get_keypoints[mask]` should be failing.

Comment (Kelvinrr) [inline autocnet/vis/graph_view.py]: yeah, I think `get_keypoints()[mask]` seems to be the way to go.

**Comment by acpaquette** [comment]
Corrected the syntax error in the get_keypoints method.

**Comment by Kelvinrr** [comment]
👍

**Pull request merged**